{"uuid": "98163227-9bce-43a7-a382-a6d3c803ac97", "children": ["c1aa71e5-ab17-4f0d-8f14-b05334bd8514"], "befores": [{"name": "_xunit_setup_method_fixture_TestSetup2", "status": "broken", "statusDetails": {"message": "KeyError: 1\n", "trace": "  File \"/Users/aymanelsayeed/.local/share/virtualenvs/AllureExample-wL7UUQNi/lib/python3.9/site-packages/pluggy/_callers.py\", line 77, in _multicall\n    res = hook_impl.function(*args)\n  File \"/Users/aymanelsayeed/.local/share/virtualenvs/AllureExample-wL7UUQNi/lib/python3.9/site-packages/_pytest/fixtures.py\", line 1123, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"/Users/aymanelsayeed/.local/share/virtualenvs/AllureExample-wL7UUQNi/lib/python3.9/site-packages/_pytest/fixtures.py\", line 895, in call_fixture_func\n    fixture_result = next(generator)\n  File \"/Users/aymanelsayeed/.local/share/virtualenvs/AllureExample-wL7UUQNi/lib/python3.9/site-packages/_pytest/python.py\", line 899, in xunit_setup_method_fixture\n    _call_with_optional_argument(func, method)\n  File \"/Users/aymanelsayeed/.local/share/virtualenvs/AllureExample-wL7UUQNi/lib/python3.9/site-packages/_pytest/python.py\", line 777, in _call_with_optional_argument\n    func(arg)\n  File \"/Users/aymanelsayeed/IdeaProjects/AllureExample/tests/tests_setup/test_steup_2.py\", line 12, in setup_method\n    Collection.pop(self.counter)\n"}, "start": 1722076912502, "stop": 1722076912502}], "start": 1722076912502, "stop": 1722076912503}